src/pages/Telemetry.vue

```diff
@@ -75,11 +75,11 @@ let robotInfo;
 
 onMounted(() => initialize());
 
-let myRos = inject<Ros>('ros');
+const roslib = useRoslibStore();
 
 function initialize() {
-  if (myRos != null) {
-    robotInfo = new RobotInfo(myRos);
+  if (roslib.ros !== null) {
+    robotInfo = new RobotInfo(roslib.ros);
```

Comment:
wouldn't it make more sense to pass in the roslib object over the ros object?

Reply:
This check is actually not needed as roslib and roslib.ros should always not be null or undefined thanks to pinia(was just a quick fix). I will refactor Telemetry in another PQ.